tests/tracer/test_logging.py

```diff
@@ -0,0 +1,284 @@
+import os
+
+
+def test_unrelated_logger_in_debug_with_patch_all(run_python_code_in_subprocess, tmpdir):
+    """
+    When importing the tracer after logging has been configured,
+                with debug mode and enabling a custom log path,
+                the ddtrace logger does not override any custom logs settings and
+                the ddtrace logger continues to have its expected configuration and
+                logs to a custom file path.
+    """
+    env = os.environ.copy()
+    env["DD_TRACE_DEBUG"] = "true"
+    log_file = tmpdir + "/testlog.log"
+    env["DD_TRACE_LOG_FILE"] = log_file
+    code = """
+import logging
+custom_logger = logging.getLogger('custom')
+custom_logger.setLevel(logging.WARN)
+
+assert custom_logger.parent.name == 'root'
+assert custom_logger.level == logging.WARN
+
+import ddtrace
+
+ddtrace.patch_all()
```

Comment:
```suggestion
```

not needed since `patch_all()` doesn't impact logging setup.

Reply:
Marking this as resolved since `ddtrace.patch_all()` has been removed from the tests.